Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+  name: "LazyPages",
+  platforms: [
+    .iOS(.v11)
+  ],
+  products: [
+    // Products define the executables and libraries a package produces, and make them visible to other packages.
```

Comment:
I think we can clean up this file a bit by removing the comments here and below

Reply:
done